README.md

```diff
@@ -0,0 +1,16 @@
+# Satellite Ansible Roles
+
+Ansible Roles to setup / configure Red Hat Satellite. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or FAM.
+
+Note: The roles here are not officially supported by Red Hat.
```

Comment:
Can we move this to the first line and make it a `#` header. I think this needs to be `clearly` stated. I would also like it if we added langugae that _encouraged_ users to consider contributing their role to foreman-ansible-modules with the incentive that will be our official vector for shipping roles in the product.

Reply:
For the second part - contributing role to foreman-ansible-modules, I will link to https://github.com/theforeman/foreman-ansible-modules/issues/646 temporarily until we define rules in FAM.